### PR TITLE
Add new study: Kocacinar_2003

### PR DESCRIPTION
**Pull request by ehwenk**
* add new study with hydraulics traits
* no contact with author
* all data transcribed from tables in the manuscript or digitised from figures

### CODE REVIEW

File: data/Kocacinar_2003/metadata.yml [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+locations:
+  University of Toronto, Ontario:
+    latitude (deg): 43.67
+    longitude (deg): -79.42
+contexts:
+- context_property: seed provenance
+  category: treatment
+  var_in: locality
+  values:
+  - value: Australian National University
```
Comment (yangsophieee) [inline data/Kocacinar_2003/metadata.yml:51]: Remove Australian National University as all of the species are from Darwin, ANU was just for one species non-native to Australia I believe

File: data/Kocacinar_2003/metadata.yml [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+    and sectioned using a sliding microtome. Sections were examined under a light
+    microscope and images analysed with imaging software (Image Pro Plus; Carsen Group
+    Inc., Markham, Ontario, Canada). The lengths of major and minor axes across a
+    conduit were averaged to obtain vessel diameters. Maximum vessel diameter (MVD)
+    was measured from each cross-section on the largest vessel. The vessels mean diameter
+    (VMD), hydraulically weighted mean diameter (HMD), mean diameter of conduits that
+    facilitate 95% of the flow (D95) and vessel frequency (VF, the number of vessels
+    per xylem area) were determined for each segment by measuring all xylem conduits
+    in three randomly chosen sectors from the cambium to pith that together represented
+    at least 20% of the stem xylem area. The hydraulically weighted mean diameter
```
Comment (yangsophieee) [inline data/Kocacinar_2003/metadata.yml:77]: Is it worth deleting the sentences from here onwards to condense the methods a bit? (And same for the rest of the traits?)

File: data/Kocacinar_2003/metadata.yml [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+    (HMD) was calculated as D = 2(sumr5/sumr4), where r is the radius of a conduit
+    (Sperry et al. 1994). Conduit efficiency value was determined by dividing the
```
Comment (yangsophieee) [inline data/Kocacinar_2003/metadata.yml:108]: Shorten methods from here?

File: data/Kocacinar_2003/metadata.yml (hunk 2 above)
Comment (ehwenk) [reply data/Kocacinar_2003/metadata.yml:77]: done

File: data/Kocacinar_2003/metadata.yml (hunk 1 above)
Comment (ehwenk) [reply data/Kocacinar_2003/metadata.yml:51]: done

File: data/Kocacinar_2003/metadata.yml (hunk 3 above)
Comment (ehwenk) [reply data/Kocacinar_2003/metadata.yml:108]: done

**Pull request merged**